bitnami/postgresql/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: postgresql
-version: 8.6.9
+version: 8.6.10
```

Comment:
You should bump minor because you are adding new features.

Reply:
Sounds good...minor bump here